services/src/main/java/org/keycloak/authentication/authenticators/client/X509ClientAuthenticator.java

```diff
@@ -179,4 +194,17 @@ public String getHelpText() {
     public String getId() {
         return PROVIDER_ID;
     }
+
+    private MtlsExtendedValidationProvider getMtlsExtendedValidationProvider(boolean mtlsExtendedValidationEnabled, ClientAuthenticationFlowContext context, ClientModel client) {
+        if (!mtlsExtendedValidationEnabled) {
+            return null;
+        } else {
+            String requiredProviderId = client.getAttribute("tls.client.certificate.extended.validation.impl");
```

Comment:
How about using `OIDCConfigAttributes.USE_MTLS_EXTENDED_VALIDATION_IMPL`?

Reply:
You're quire right here. Adding static naming, and not using it. fixed